primitives/src/opcodes.rs

```diff
@@ -565,6 +565,12 @@ mod tests {
         assert_eq!(s, "    OP_NOP");
     }
 
+    #[test]
+    fn ordinary_op_code() {
+        let ordinary_op = Ordinary::from_opcode(OP_PUSHDATA1).expect("0x4C");
+        assert_eq!(ordinary_op.to_u8(), 0x4C_u8);
```

Comment:
Just for educational purposes; the `_u8` is not needed on `0x4C` here. I think it is because the compiler can infer the type from the type of `ordinary_op.to_u8()`.

Reply:
Thanks, I just assumed I had to cast to a u8